libs/chlorophyll/scss/components/accordion/_mixins.scss

```diff
@@ -89,4 +90,20 @@
       }
     }
   }
+
 }
+
+@mixin accordion-list-item-content() {
+  overflow: hidden;
+  transition: height 200ms linear, opacity 200ms linear;
```

Comment:
Since we need to use the hidden attribute, animations will be hard to implement. So we'll have to leave them for now and prioritise accessibility.

Reply:
Based on your comments, I removed the accordion-list-item-content style all together since it did not add anything.